core/src/main/java/org/apache/iceberg/SnapshotManager.java

```diff
@@ -22,6 +22,7 @@
 
 public class SnapshotManager implements ManageSnapshots {
 
+  private final boolean isTransactionExternal;
```

Comment:
Nit naming: could we call it "isExternalTransaction" or even just "externalTransaction"

Reply:
Renamed this to isExternalTransaction